doc/docs/Python_User_Interface.md

```diff
@@ -287,13 +286,6 @@ Python. `Vector3` is a `meep` class.
   the minimum refractive index (usually 1), and in practice $S$ should be slightly
   smaller.
 
-+ **`loop_tile_base_db`, `loop_tile_base_eh` [`number`]** — To improve the [memory locality](https://en.wikipedia.org/wiki/Locality_of_reference)
-  of the field updates, Meep has an experimental feature to "tile" the loops over the Yee grid
-  voxels. The splitting of the update loops for step-curl and update-eh into tiles or subdomains
-  involves a recursive-bisection method in which the base case for the number of voxels is
-  specified using these two parameters, respectively. The default value is 0 or no tiling;
-  a typical nonzero value to try would be 10000.
-
```

Comment:
Looks like a rebasing problem?

Reply:
Yeah. Please ignore this PR - Recreated this change in a  fresh branch
See https://github.com/NanoComp/meep/pull/1784